newIDE/app/src/Profile/CurrentUsageDisplayer.js

```diff
@@ -10,70 +10,145 @@ import {
 import PlaceholderLoader from '../UI/PlaceholderLoader';
 import Text from '../UI/Text';
 import GetSubscriptionCard from './Subscription/GetSubscriptionCard';
-import AlertMessage from '../UI/AlertMessage';
 import { ColumnStackLayout } from '../UI/Layout';
+import { Column, Line } from '../UI/Grid';
+import GDevelopThemeContext from '../UI/Theme/GDevelopThemeContext';
+
+const styles = {
+  subscriptionContainer: {
+    display: 'flex',
+    borderRadius: 10,
+    alignItems: 'center',
+  },
+  diamondIcon: {
+    width: 50,
+    height: 50,
+  },
+};
 
 type Props = {|
   subscription: ?Subscription,
   quota: ?Quota,
+  numberOfPendingBuilds: number,
   onChangeSubscription: () => void,
 |};
 
 const CurrentUsageDisplayer = ({
   subscription,
   quota,
+  numberOfPendingBuilds,
   onChangeSubscription,
 }: Props) => {
-  if (!quota) return <PlaceholderLoader />;
+  const gdevelopTheme = React.useContext(GDevelopThemeContext);
+  if (!quota || !subscription) return <PlaceholderLoader />;
+  const isFeatureLocked = quota.max === 0;
   const hasSubscription = hasValidSubscriptionPlan(subscription);
-  const loadedButHasNoSubscription =
-    subscription && !hasValidSubscriptionPlan(subscription);
-  const remainingBuilds = Math.max(quota.max - quota.current, 0);
+  const usedBuilds = Math.min(quota.current + numberOfPendingBuilds, quota.max);
+  const remainingBuilds = Math.max(quota.max - usedBuilds, 0);
+  const usageRatio = `${usedBuilds}/${quota.max}`;
   const remainingMultipleMessage = (
     <Trans>
-      You have {remainingBuilds} builds remaining (you have used {quota.current}{' '}
-      out of {quota.max} in the last 24h).
+      You have {remainingBuilds} builds remaining (You have used {usageRatio} in
```

Comment:
Do we need to capitalize the first letter in parentheses?

Reply:
no my bad